### PR TITLE
Update flake8. NFC.

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: tests/test_other.py [hunk 1]
```diff
@@ -558,11 +558,12 @@ def test_cmake(self, test_dir, output_file, cmake_args):
...
-                      'NMake Makefiles'     : {'build'   : ['nmake', '/NOLOGO']}, # noqa
-                      'Unix Makefiles'      : {'build'   : ['make']}, # noqa
-                      'Ninja'               : {'build'   : ['ninja']}, # noqa
-                      'Eclipse CDT4 - Ninja': {'build'   : ['ninja']}, # noqa
+    configurations = {
+      'MinGW Makefiles'     : {'build'   : ['mingw32-make'] }, # noqa
+      'NMake Makefiles'     : {'build'   : ['nmake', '/NOLOGO']}, # noqa
+      'Unix Makefiles'      : {'build'   : ['make']}, # noqa
+      'Ninja'               : {'build'   : ['ninja']}, # noqa
+      'Eclipse CDT4 - Ninja': {'build'   : ['ninja']}, # noqa
```
Comment (tlively) [inline tests/test_other.py:566]: Does the new flake8 prefer this format?

File: .flake8 [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = E111,E114,E501,E261,E266,E121,E402,E241,W504
+ignore = E111,E114,E501,E261,E266,E121,E402,E241,W504,E741
```
Comment (tlively) [inline .flake8:2]: Do we have a lot of variables named `I`, `l`, or `O`? If there are not too many, it might be good to rename them.

Comment (sbc100) [reply .flake8:2]: Ah yes, we have a lot of `[l for l in lines ...]` comprehensions that triggers this.. and I think that is a little overzelous.

Comment (sbc100) [reply .flake8:2]: I guess we could change to `[ln for ln in lines ..]`?

Comment (tlively) [reply .flake8:2]: oh yeah, I do like having single-letter variables in comprehensions. It's too bad we can't apply this lint everywhere except in comprehensions. I'm fine either way.

**Review by tlively — APPROVED** [review]

**Pull request merged**